resources/setup.sh

```diff
@@ -10,7 +10,7 @@ echo $STATIC_IP > /var/kube-config/static-ip
 docker info
 
 # add deps
-apk add --update sudo curl ca-certificates bash less findutils supervisor tzdata socat lz4 conntrack-tools
+apk add --update sudo curl ca-certificates bash less findutils supervisor tzdata socat lz4 conntrack-tools git
```

Comment:
This is specific to your needs i think?

Reply:
Happy to remove if you decide that Kind does not need support for csi-snapshotting just yet

Reply:
intent of kind is to allow testing of kube deployed stuff in CI etc, volume snapshotting sits pretty far outside that remit, happy to support people to add via after-cluster script, but don't think it needs to be in upstream images :+1: